### PR TITLE
Added IT Flex Banners

### PR DESCRIPTION
**Pull request by JustinDoan**

### CODE REVIEW

File: modals/US/ni.json [hunk 1]
```diff
@@ -38,8 +38,7 @@
                 "No interest will be charged on the purchase if you pay it off in full within 6 months. If you do not, interest will be charged on the purchase from the purchase date at the Purchase APR applicable to your account.",
                 "For New Accounts: Variable Purchase APR is {aprEntry.apr}%. The APR is accurate as of {aprEntry.formattedDate} and will vary with the market based on the Prime Rate (as defined in your credit card agreement). Minimum interest charge is $2.00.",
                 "Individual items that are less than $99 qualify for special financing when combined for a total of $99 or more in a single transaction.",
-                "Multiple separate transactions of less than $99 per transaction cannot be combined to meet the minimum purchase amount.",
-                "Special Financing is not available when using the PayPal Send Money feature to make a purchase."
+                "Multiple separate transactions of less than $99 per transaction cannot be combined to meet the minimum purchase amount."
```
Comment (rosman21) [inline modals/US/ni.json:41]: why are these changes in this branch?

File: modals/US/gpl.json [hunk 2]
```diff
@@ -37,11 +37,12 @@
...
-                "When applying, a soft credit check may be needed, but will not affect your credit score.",
-                "You must be of legal age in your U.S. state of residence to use Pay in 4.",
-                "Loans to California residents are made or arranged pursuant to a California Finance Lenders Law license.",
-                "PayPal, Inc. is a Georgia Installment Lender Licensee, NMLS #910457."
+                "Pay in 4 is not currently available to residents of Missouri, Nevada, New Mexico, North Dakota, Wisconsin, or any U.S. Territories. Offer availability depends on the merchant and also may not be available for certain recurring, subscription services.​",
+                "When applying, a soft credit check may be needed, but will not affect your credit score.​",
+                "You must be of legal age in your U.S. state of residence to use Pay in 4.​",
+                "Loans to California residents are made or arranged pursuant to a California Financing Law License.​",
```
Comment (rosman21) [inline modals/US/gpl.json:43]: even this one?

File: modals/US/ni.json (hunk 1 above)
Comment (JustinDoan) [reply modals/US/ni.json:41]: Looks like the branch im merging into might be behind develop, since i've merged develop into this branch

Comment (rosman21) [reply modals/US/ni.json:41]: yea that makes sense. I was confused at first. I guess when it is ready for a full review just merge develop in first

Comment (JustinDoan) [reply modals/US/ni.json:41]: Weird i merged in develop to this and the gpl-it branch but these changes still appear to be here.

Comment (JustinDoan) [reply modals/US/ni.json:41]: Will look more into this

**Pull request closed without merging**